Gemfile

```diff
@@ -87,7 +87,7 @@ gem 'redis', '>= 4.0', require: ['redis', 'redis/connection/hiredis']
 gem 'sidekiq'
 gem 'sidekiq-scheduler'
 
-gem "cable_ready", "5.0.0.pre2"
+gem "cable_ready", "5.0.0.pre3"
```

Comment:
I would find an explanation helpful in the commit message.

Reply:
Added :+1:

The new version contains the `redirect_to` operation, which was needed here for Stripe and Paypal.